src/components/Sidebar/AddChoiceButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import styles from './Sidebar.module.scss';
+import { Button } from '@material-ui/core';
+
+interface AddChoiceButtonProps {
+  addChoiceNode: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+// TODO: in PATHWAYS-256 Add choice needs to update the pathway with the new transition and state
+const AddChoiceButton: FC<AddChoiceButtonProps> = ({ addChoiceNode }) => {
+  return (
+    <table>
```

Comment:
In other places we use the Material UI Table, should we be using that here as well for consistency?

Reply:
Yes.

Reply:
Actually, I don't think a table is even needed here. We can get by using divs and flexbox.